tensorboard/webapp/widgets/histogram/histogram_v2_component.ts

```diff
@@ -55,83 +77,178 @@ export class HistogramV2Component implements OnChanges {
 
   @Input() data!: HistogramData;
 
-  private yScale?:
-    | d3.ScaleTime<number, number>
-    | d3.ScaleLinear<number, number>;
+  readonly HistogramMode = HistogramMode;
 
-  constructor(private readonly host: ElementRef) {}
+  private layout: Layout = {
+    histogramHeight: 1,
+    contentClientRect: {height: 1, width: 1},
```

Comment:
Just to confirm: is it important for these to be nonzero, or is `1` just an arbitrary default value?  I often use `0` as default myself, and I'm not sure from reading whether there might be something wrong about choosing `0` here.